r/noteable/requirements.R

```diff
@@ -1,5 +1,7 @@
 install.packages('reticulate', repos='http://cran.us.r-project.org')
 library(reticulate)
+# for whatever reason, reticulate doesn't work with the default python installation
+install_miniconda()
```

Comment:
Unlike the portions of `micromamba` that were removed as other parts of this PR, only `reticulate` uses this to establish its virtual environment, and it does not negatively impact package installs from non-root users